packages/integrations/mdx/package.json

```diff
@@ -39,6 +39,7 @@
     "github-slugger": "^1.4.0",
     "gray-matter": "^4.0.3",
     "kleur": "^4.1.4",
+    "rehype-pretty-code": "^0.4.0",
```

Comment:
Should this have been a dev dependency as it’s only used in the test?

Reply:
Oh right. I used the wrong install command. Fixing this now, thanks for catching this! 🙏